### DISCUSSION TITLE
How would you go on keeping track of the time after someone said a command?

### DISCUSSION THREAD

**Discussion by sirquackiii** [Rapptz/discord.py#6737] [Q&A]
So, I'm working on a simple economy discord bot, and for the "daily" command, where you get your daily amount of coins, I need a way to check if someone has already gotten their daily coins. I can obviously do this with `time.sleep(x)`, but that would stop the whole program from listening to commands. How can I keep track of time while still having commands be useable?

Thanks!

**Comment by MrKomodoDragon** [comment]
If you want to sleep, you could use `await asyncio.sleep(x)`

**Comment by Luc1412** [comment]
You could save the latest timestamp. If the timedelta between now and the latest daily timestamp is smaller than 1 you can perform the action. 
If you need to execute an action instantly you can throw it into a tasks.Loop
